website/docs/basics/blocks-wrapper.md

```diff
@@ -22,7 +24,7 @@ This attribute controls if you are going to see options in the block editor for
 
 ### wrapperUseSimple
 
-We wrapped some of the options in a specific condition, and we call it `wrapperUseSimple`. In general, this attribute is set to `true` when you only want simplified options on your block. Natively it is used inside all of the inner blocks in the column block. Generali in all of the inner blocks, it is a good idea to simplify the wrapper.
+We wrapped some of the options in a specific condition, and we call it `wrapperUseSimple`. In general, this attribute is set to `true` when you only want simplified options on your block. Natively, it is used inside all of the inner blocks in the column block. Because we don't need wrappers inside wrappers inside wrappers (and so on). It's a good rule of thumb to use a simple wrapper in all of the inner blocks.
```

Comment:
Maybe add a link to `dispatch` example. Something like "Click here to see how to force simpleWrapper in all innerBlocks.